src/main.rs

```diff
@@ -258,6 +260,13 @@ fn main() {
             }
         }
 
+        "clean" => {
+            match fs::remove_dir_all(&config.dest) {
```

Comment:
hmm weird. cobalt *does* create the dir for me.

https://gist.github.com/rjgoldsborough/08fc7700a847b9318a7ebd1da10ade89